zebra-chain/src/block/merkle.rs

```diff
@@ -115,7 +183,32 @@ where
     }
 }
 
+impl std::iter::FromIterator<UnminedTx> for Root {
```

Comment:
Should the new `UnminedTx` conversion for `merkle::Root` be behind a feature flag?

Reply:
I think this is okay without a feature flag too, it's not specific to the new feature or affecting any existing features